frontstage/controllers/party_controller.py

```diff
@@ -379,34 +360,26 @@ def caching_case_data(cache_data, business_ids, tag):
         thread.join()
 
 
-def get_survey_list_details_for_party(enrolment_data: dict, tag: str, business_party_id: str, survey_id: str):
+def get_survey_list_details_for_party(respondent_enrolments: list, tag: str, business_party_id: str, survey_id: str):
```

Comment:
Again historical, but it would be nice to rename this method now we've already got the surveys in the enrolment data. Eyeballing it I was wondering why we were going back to get surveys but its actually grabbing the cases to be displayed and not the surveys.